draft-ietf-tls-tls13.md

```diff
@@ -1383,10 +1383,7 @@ Structure of this message:
 
 %%% Key Exchange Messages
 
-       struct {
-           uint8 major;
-           uint8 minor;
-       } ProtocolVersion;
+       uint8 ProtocolVersionID[2];
```

Comment:
The point of the change is to formally recognize that these are not version *numbers*, but simply *versions*.  That is, incompatible manifestations of a protocol that serve the same approximate function.  Or, in other words, version numbers failed us, so we're trying this instead.